### PR TITLE
CLOUD-1255: name change and typo fixes

### PR DESCRIPTION
**Pull request by StevePlace68**
-Change all instances of "onprem" or "on-prem" to "Launchpad" 
-Fix typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,38 +15,38 @@ the Stardog server itself, a user can access their Stardog server via:
...
 ## How it Works
 
-There are 2 main flows (Kerberos being the exception) used by the On-Prem application with respect to authentication.
+There are 2 main flows (Kerberos being the exception) used by the Launchpad application with respect to authentication.
 
 1. Basic Auth (Stardog username and password)
 
 2. Identity Provider (e.g. Google)
 
-Both authentication flows utilize Stardog's [OAuth 2.0 Integration](https://docs.stardog.com/operating-stardog/security/oauth-integration). In short, Stardog can produce JWT tokens it can then accept for authenticated API requests. Stardog can also be configured to accept JWT tokens issued by a trusted issuer and optionally auto-create users if roles from the IdP are mapped properly to pre-defined Stardog roles.
```
Comment (paulplace) [inline README.md:24]: Most readers of this doc likely already know that "JWT" stands for "JSON Web Token", but for something techie like this, I like to define the abbreviation before using it throughout the rest of the document. In fact, once you've made clear that the "T" in "JWT" means "token", you can use "token" in places were it makes the prose clearer.

File: README.md [hunk 2]
```diff
@@ -15,38 +15,38 @@ the Stardog server itself, a user can access their Stardog server via:
...
-  On-Prem->>IdP: Successful user authentication
-  IdP->>On-Prem: IdP JWT returned
-  Note over IdP,On-Prem: On-Prem saves profile information <br> contained in IdP JWT in a session and discards it.
-  On-Prem->>Stardog: Stardog API requests with On-Prem JWT
-  Note over On-Prem,Stardog: On-Prem generates its JWTs Stardog server <br> is configured to accept using information contained in the cookie.
+  Launchpad->>IdP: Successful user authentication
+  IdP->>Launchpad: IdP JWT returned
+  Note over IdP,Launchpad: Launchpad saves profile information <br> contained in IdP JWT in a session and discards it.
+  Launchpad->>Stardog: Stardog API requests with Launchpad JWT
+  Note over Launchpad,Stardog: Launchpad generates its JWT's Stardog server <br> is configured to accept using information contained in the cookie.
```
Comment (paulplace) [inline README.md:44]: Over the years, I have tried both the `s` and `'s` versions of pluralizing an abstract object. I have reached a point of preferring not to use an apostrophe except when the name of the object being pluralized ends with an `s`.

I don't have a strong enough opinion to insist that you go back and change them all in this PR, but now I have said what I have to say.

**Review by paulplace — APPROVED** [review]
Looks good. I made a couple suggestions, but neither is a show stopper.

File: README.md (hunk 1 above)
Comment (StevePlace68) [reply README.md:24]: I don't want to go through all the instances of JWT and figure out where it does/doesn't make sense to change it to token, but I liked the idea of spelling "JSON Web Tokens" enough that I did so at the first instance of JWT in the main readme.

File: README.md (hunk 2 above)
Comment (StevePlace68) [reply README.md:44]: This was easy enough to find+replace safely (because of the apostrophe) that I did so.

**Pull request merged**